en/PlanView/PlanView.md

```diff
@@ -1,98 +1,140 @@
+> **Tip** The Fly View displays the *actual* home position set by the vehicle firmware when it arms (this where the vehicle will return in Return/RTL mode). 
 
-### Mission Settings
-The Mission Settings panel allows you to specify values which apply to the entire mission, or settings you want to control right at the 
-beginning of a mission. This is the first item in the mission list on the right of the screen. For example, you can change the altitude for all 
-mission waypoints to the value you set in "Waypoint alt".
+<img src="../../images/plan/MissionSettingsPlannedHomePositionSection.jpg" style="width: 300px;"/>
 
-<img src="../../images/plan/MissionSettings.png" style="width: 200px;"/>
 
+## Mission Command List {#mission_command_list}
 
-## Mission Display
+Mission commands for the current mission are listed on the right side of the view. You can select individual items to edit their values. 
+Above are a set of options to switch between editing the mission, GeoFence and rally points.
 
-In the center of the map you will see a visualization of your current mission. You can click on the indicators to select them and then you can 
-also drag them around to reposition them. 
 
-![](../../images/plan/PlanToolbar.png)
+### Mission Command Editors
+
+Click on a mission command in the list to display its editor (in which you can set/change the command attributes).
+
+You can change the type of the command by clicking on the command name (for example: "Waypoint"). 
+This will display the *Select Mission Command* dialog shown below. To the right of each command name is a menu that you can click to access to additional options such as *Insert* and *Delete*.
+
+<img src="../../images/plan/MissionCommands.png" style="width: 200px;"/>
+
+The list of commands displayed in the dialog can be filtered by category. For example, to see all commands, choose **All commands** from **Category** drop down menu. 
 
-At the top of the *Plan View* you will see the Plan Toolbar which shows information for the currently selected waypoint relative to the previous 
-waypoint as well as statistics for the entire mission. For e.g. to the left you will find altitude difference and distance from the previous 
-waypoint, whereas to the right you will find an estimate of the horizontal distance and time taken by the total mission. "Max telem dist" is the 
-distance between the Planned Home position (where your GCS is expected to be) and the furthest waypoint.
+> **Note** The list of available commands will depend on firmware and vehicle type. Examples may include: Waypoint, Start image capture, Jump to item (to repeat mission) and other commands.
 
-![](../../images/plan/MissionHeightDisplay.png)
 
-At the bottom of the map you will see a representation of the height differences between your mission commands.
+### Mission Settings
+
+The *Mission Settings* (Mission Start) panel is the first item in the mission command list (right of the screen). 
```

Comment:
Mission Start is grab bag of things. Some defaults, some things which affect the beginning of the mission, some which affect the end. It is not really the first item in the command list. Since if you leave things at defaults it will not change anything with respect to the start of the mission. And it also affects the end of the mission.

**Waypoint alt**
* Set the default altitude for the first item added to a plan
* The altitude for subsequent items is the same as the item previous to the one added.
* If you changed this and there are items in the plan it should also prompt you to change the altitude of all items in the plan to this new altitude. (If that isn't happening file an Issue).

Reply:
Thanks. I created defect report https://github.com/mavlink/qgroundcontrol/issues/6437 . Will update this part of the doc shortly.